data/language/dutch.txt

```diff
@@ -66,7 +66,7 @@ STR_0064    :Attractie
 STR_0065    :Omgekeerde monorail
 STR_0066    :Attractie
 STR_0067    :Attractie
-STR_0068    :Heartline-twisterachtbaan
+STR_0068    :Heartline-Twisterachtbaan
```

Comment:
Eens. Heb het gedaan omdat er eerder ook Heartline-Twisterachtbaan stond, maar had dat natuurlijk moeten veranderen in Heartline-twisterachtbaan.